src/Util/MailerLoggerUtil.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace App\Util;
+
+use Symfony\Bridge\Monolog\Logger;
+
+use Psr\Log\LogLevel;
+use Swift_Events_SendEvent;
+use Swift_Events_SendListener;
+
+
+
+/**
+ * A utility to log Swift e-mails.
+ */
+class MailerLoggerUtil implements Swift_Events_SendListener
+{
+     /**
+      * A Monolog logger.
+      *
+      * @var Logger
+      */
+    protected $logger;
+
+    /**
+     * MailerLoggerUtil constructor.
+     *
+     * @param LoggerInterface $logger The Logger Interface.
+     */
+    public function __construct(Logger $logger)
```

Comment:
I don't know how I made the other loggers work with Monolog\Logger class injection, but according to the documentation whatever you did was right . Should use the LoggerInterface from Psr\Log
[https://symfony.com/doc/current/logging/channels_handlers.html#how-to-autowire-logger-channels](url)